backend/gncitizen/core/taxonomy/routes.py

```diff
@@ -110,6 +281,119 @@ def get_taxon_from_cd_nom(cd_nom):
     """
     """Renvoie la fiche TaxRef de l'espèce d'après le cd_nom"""
     try:
-        return get_specie_from_cd_nom(cd_nom=cd_nom)
+        return get_taxa_by_cd_nom(cd_nom=cd_nom)
     except Exception as e:
         return {"message": str(e)}, 400
+    
+
+@taxo_api.route("/taxonomy/tmedias/types", methods=["GET"])
+@json_resp
+def get_media_types()-> List[Dict[str, Union[int, str]]]:
+  """Get all media types.
+    ---
+    tags:
+     - Taxon
+    responses:
+      200:
+        description: A list of all media types.
+        content:
+          application/json:
+            schema:
+              type: array
+              items:
+                type: object
+                properties:
+                  id_type:
+                    type: integer
+                    description: Unique identifier for the media type.
+                  nom_type_media:
+                    type: string
+                    description: Name of the media type.
+                  desc_type_media:
+                    type: string
+                    description: Description of the media type.
+      400:
+        description: Bad request.
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                message:
+                  type: string
+                  description: Error message.
+  """
+  try:
+      return get_all_medias_types()
+  except Exception as e:
+      return {"message": str(e)}, 400 
+
+@taxo_api.route("/taxonomy/bibattributs", methods=["GET"])
```

Comment:
C'est donc pas exclusivement géré côté frontend ?

Reply:
ce sont uniquement la définition des routes qui sont coté backend . Ca permet de garder une homogénéité dans la gestion des routes API (notamment pour le renvoi des erreurs) . Par contre , les modifications n'utilisent plus la mise en cache coté backend (ce qui était le cas pour les taxhub_list) . Ce qui a été dit lors de la réunion, proposé et expliqué par Amandine c'était d'avoir une mise en cache des types de médias et attributs coté frontend (et c'est le cas dans la PR , voir le globalCacheService) .